Lib/asyncio/proactor_events.py

```diff
@@ -232,8 +232,9 @@ class _ProactorBaseWritePipeTransport(_ProactorBasePipeTransport,
 
     def write(self, data):
         if not isinstance(data, (bytes, bytearray, memoryview)):
-            raise TypeError('data argument must be byte-ish (%r)',
-                            type(data))
+            msg = ("data argument must be a bytes-like object, not '%s'" %
```

Comment:
Why not use string tuples for ``format`` like this?

```py
            msg = ("data argument must be a bytes-like object, not '{}'"
            ).format(data)
```

also are you missing an

```py
            raise TypeError(msg)
```
anywhere?